app/org/sagebionetworks/bridge/services/UserAdminServiceImpl.java

```diff
@@ -154,16 +146,4 @@ private boolean deleteUserInStudy(Study study, Account account, User user) throw
             return false;
         }
     }
-
-    private void removeAllHealthDataRecords(Study study, User user) throws BridgeServiceException {
```

Comment:
We have the requirement to delete everything of a user when we delete a user. I mean, a regulatory requirement to do this. We have to have that functionality somewhere... do we?